packages/core/src/workflows/workflow.ts

```diff
@@ -242,7 +235,7 @@ export class Workflow<
    * @returns this instance for method chaining
    */
   commit() {
-    this.#validateWorkflow();
+    // this.#validateWorkflow();
```

Comment:
dont we still need to validate on workflow commit?

Reply:
yeah but not how we did previously

I'll open a follow-up PR for validating circular dependencies
I think that's all we'd need for this new structure

Reply:
alrighty, sounds good